apps/frontend/src/components/Auth/apiClient.tsx

```diff
@@ -110,9 +111,22 @@ export class ApiClient {
       },
     ];
   }
-
   //TODO: hook up to backend
   public async saveComment(comment: string, timesheetID: number): Promise<Boolean> {
```

Comment:
lets just delete this

Reply:
Yep, I think the `saveComment` function can be wiped for now

Reply:
9150d662dbe36ac1de9e195eb7c626f27494a037